.travis.mk

```diff
@@ -317,6 +317,22 @@ test_oos_build:
 		-i ${DOCKER_IMAGE_TARANTOOL} \
 		make -f .travis.mk ${OOS_BUILD_RULE}
 
+# Odroid arm64
+
+deps_odroid:
+	sudo apt update -y && sudo apt -y install git build-essential cmake make zlib1g-dev \
+		libreadline-dev libncurses5-dev libssl-dev libunwind-dev libicu-dev python3 \
+		python3-six python3-gevent python3-pip
+
+build_odroid:
```

Comment:
Side note: I guess we need to file an issue (at least) to clean up .travis.mk. I have no idea why you need a separate rule for odroid except the fact `configure_debian` has hardcoded flags. But this doesn't relate to the issue of course, just my whining.

Reply:
We have strong opinion that we must provide for each testing configuration separate steps like bootstrap, build, test and etc. Anyway this style can be discussed and corrected within new issue.